contrib/babelfishpg_tsql/src/tsql_for/forjson.c

```diff
@@ -15,21 +9,55 @@
 #include "parser/parser.h"
 #include "utils/builtins.h"
 #include "utils/json.h"
+#include "utils/jsonb.h"
 #include "utils/syscache.h"
 #include "utils/typcache.h"
+#include "utils/hsearch.h"
 #include "catalog/pg_type.h"
 #include "catalog/namespace.h"
 
 #include "tsql_for.h"
 
-static void tsql_row_to_json(StringInfo state, Datum record, bool include_null_values);
+#define TABLE_SIZE 100
+
+// For holding information regarding the state of the FOR JSON call
```

Comment:
This comment is redundant with the name of the struct - you can either remove this or expand on why this struct is needed if it is not obvious